### PR TITLE
RATIS-1421. [Streaming]make client NioEventLoopGroup size configurable.

### PR DESCRIPTION
**Pull request by captainzmc**
## What changes were proposed in this pull request?

See: https://issues.apache.org/jira/browse/RATIS-1421

### CODE REVIEW

**Comment by captainzmc** [comment]
hi @szetszwo @bshashikant , can you help review this patch?

File: ratis-netty/src/main/java/org/apache/ratis/netty/NettyConfigKeys.java [hunk 1]
```diff
@@ -68,6 +69,18 @@ static int port(RaftProperties properties) {
     static void setPort(RaftProperties properties, int port) {
       setInt(properties::setInt, PORT_KEY, port);
     }
+
+    String EVENT_LOOP_THREADS_KEY = NettyConfigKeys.PREFIX + ".client.eventLoopThreads";
```
Comment (szetszwo) [inline ratis-netty/src/main/java/org/apache/ratis/netty/NettyConfigKeys.java:73]: It should be just PREFIX.  Also, let's add "client" to all the names, i.e.
```
    String CLIENT_EVENT_LOOP_THREADS_KEY = PREFIX + ".client.eventLoopThreads";
    int CLIENT_EVENT_LOOP_THREADS_DEFAULT = Math.max(1, NettyRuntime.availableProcessors() * 2);

    static int clientEventLoopThreads(RaftProperties properties) {
      return getInt(properties::getInt,
          CLIENT_EVENT_LOOP_THREADS_KEY, CLIENT_EVENT_LOOP_THREADS_DEFAULT, getDefaultLog(), requireMin(1), requireMax(65536));
    }

    static void setClientEventLoopThreads(RaftProperties properties, int eventLoopThreads) {
      setInt(properties::setInt, EVENT_LOOP_THREADS_KEY, eventLoopThreads);
    }
```

**Review by szetszwo — COMMENTED** [review]
The change looks good.  Just a comment on the names.  Thanks.

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**